rest/importing-content-model/AddType.curl

```diff
@@ -4,48 +4,64 @@ curl--request POST\
   --header 'Authorization: Bearer <YOUR_MANAGEMENT_API_KEY>'\
   --header 'Content-type: application/json'\
   --data ' {
-"name": "Blogpost",
-"codename": "blogpost",
-"content_groups": [{
-    "name": "Blog content",
-    "external_id": "blog_content"
-  },
-  {
-    "name": "Author",
-    "external_id": "author"
-  }
-],
-"elements": [{
-    "name": "Title",
-    "type": "text",
-    "content_group": {
-      "external_id": "blog_content"
-    }
-  },
-  {
-    "name": "Image",
-    "type": "asset",
-    "content_group": {
-      "external_id": "blog_content"
-    }
-  },
-  {
-    "name": "Blog content",
-    "type": "rich_text",
-    "content_group": {
-      "external_id": "blog_content"
-    }
-  },
-  {
+ "name": "Blogpost",
+ "codename": "blogpost",
+ "content_groups": [
+   {
+     "name": "Content",
+     "external_id": "content"
+   },
+   {
+     "name": "Metadata",
+     "external_id": "metadata"
+   },
+   {
+     "name": "Topic",
+     "external_id": "topic"
+   }
+ ],
+ "elements": [
+   {
+     "name": "Title",
+     "type": "text",
+     "content_group": {
+       "external_id": "content"
+     }
+   },
+   {
+     "name": "Image",
+     "type": "asset",
+     "content_group": {
+       "external_id": "content"
+     }
+   },
+   {
+     "name": "Blog content",
+     "type": "rich_text",
+     "content_group": {
+       "external_id": "content"
+     }
+   },
+   {
     "snippet": {
-      "codename": "author"
+        "codename": "metadata"
     },
     "type": "snippet",
-    "id": "b9f801b8-024b-461a-a8fa-bd213e46edc1",
-    "codename": "author_profile",
+    "codename": "metadata",
+    "content_group": {
+        "external_id": "metadata"
+		}
```

Comment:
Compared to `content_group` a few lines below, the closing bracket seems out of place.